curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/657efa642593c5746acc5c81.md

```diff
@@ -7,15 +7,13 @@ dashedName: step-5
 
 # --description--
 
-Inside the `if` statement body, you need to convert any uppercase character to lowercase and prepend an underscore to this lowercase character.
-
-Use the `.lower()` string method to convert uppercase characters to lowercase characters. You can then concatenate an underscore to the character using the plus sign.
+You'll need to transform uppercase characters into lowercase and add an underscore at the beginning of each converted character. Then concatenate an underscore to the character using the plus sign.
```

Comment:
I think, we should not explicitly tell the campers how to modify the string.

An updated instruction could look like this:

```
Inside the `if` statement body, you need to convert any uppercase character to lowercase and prepend an underscore to this lowercase character.

Use the `.lower()` string method to convert uppercase characters to lowercase characters. Then, prepend an underscore to the character using concatenation. Assign the results to a variable named `converted_character`.
```

For this instructions, the hints would also be modified.

Reply:
In addition to the entire description being replaced do we need to get rid of the sample code too?